packages/block-library/src/post-comments/index.php

```diff
@@ -26,7 +26,22 @@ function render_block_core_post_comments( $attributes, $content, $block ) {
 	setup_postdata( $post );
 
 	ob_start();
+	// There's a deprecation warning generated by WP Core.
+	// Ideally this deprecation is removed from Core.
+	// In the meantime, this removes it from the output.
+	add_filter(
+		'deprecated_file_trigger_error',
+		function() {
+			return false;
+		}
```

Comment:
Minor point, but you can use `__return_false` instead.

Reply:
😆 Looks like we were thinking the same and I missed your comment by a few seconds before I hit the publish button on my review